universal-application-tool-0.0.1/app/modules/CloudStorageModule.java

```diff
@@ -52,6 +54,9 @@ protected void configure() {
   }
 
   private void bindCloudStorageStrategy(String storageProvider) {
+    // Defaulting to this for now because prod file upload to Azure blob storage hasn't been
+    // implemented yet.
+    bind(FileUploadStrategy.class).to(AwsFileUploadStrategy.class);
```

Comment:
suggestion (blocking): add TODO with reference to GH issue

Reply:
Done.